cmd/hz/app/app.go

```diff
@@ -44,6 +45,15 @@ func New(c *cli.Context) error {
 	setLogVerbose(args.Verbose)
 	logs.Debugf("args: %#v\n", args)
 
+	exist, err := util.PathExist(path.Join(args.OutDir, meta.ManifestFile))
+	if err != nil {
+		return cli.Exit(err, meta.LoadError)
+	}
+
+	if exist {
+		return cli.Exit(fmt.Errorf("project already exists"), meta.LoadError)
```

Comment:
”project already exists“ is a little bit simple / maybe add some instruction for users(tell why/how)?

Reply:
ok

Reply:
add a error info "If you still want to "new" a project please delete the ".hz" manually

Reply:
thanks